contracts/swaps/SwapsUser.sol

```diff
@@ -91,7 +95,8 @@ contract SwapsUser is State, SwapsEvents, FeesHelper {
         uint256 sourceTokenAmountUsed;
 
         uint256 tradingFee;
-        if (!miscBool) { // bypassFee
+        if (!miscBool) {
```

Comment:
what is miscBool probably need other name for this var

Reply:
it is bypassFee I believe. I did not modify the naming of this